src/stuf/stuf_bg/models.py

```diff
@@ -12,7 +12,7 @@ class StufBGConfig(SingletonModel):
     """
 
     service = models.OneToOneField(
-        "stuf.SoapService",
+        "stuf.StufService",
```

Comment:
this is a new model, for which a new table gets created. in https://github.com/open-formulieren/open-forms/pull/735/files#diff-7accd7042730691e91ee089026cb8ed2b1d16374a45e925f61d819db039e7a2aR16, you copy the data over, but there is no guarantee that the primary keys will stay the same.

I think it'd be wiser to do this migration in three steps:

1. rename the `service` field to `old_service` and add the field `new_service` pointing to the new model
2. data migration -> linking the correct new service
3. dropping the `old_service` field and renaming `new_service` to `service`